config.yaml

```diff
@@ -121,17 +103,24 @@
             id: syn13363443
             format: csv
         final_format: json
-        custom_transformations: 1
+        custom_transformations:
+          adjusted_p_value_threshold: 0.05
+          protein_level_threshold: 0.05
+        column_rename:
+          ensg: ensembl_gene_id
         provenance:
-          - syn26868788
+          - syn25953363
+          - syn12514826
+          - syn12514912
+          - syn18689335
+          - syn35221005
+          - syn27211942
           - syn12540368
           - syn27211878
           - syn13363443
         agora_rename:
-          eqtl: haseqtl
-          igap: isIGAP
+          symbol: hgnc_symbol
```

Comment:
Same comment as for test_config.yaml - include the eqtl and igap agora_renames

Reply:
See above. Addressed in commit [5c5dc71e9f5b92ab8b226faaf2877e8a35360b17](https://github.com/jaclynbeck-sage/agora-data-tools/commit/5c5dc71e9f5b92ab8b226faaf2877e8a35360b17).

Reply:
Your comment about deleting AG-469_config.yaml won't let me comment in reply, so commenting here. Correct, we don't need that config file anymore. I've deleted it from the repository in commit [5c5dc71e9f5b92ab8b226faaf2877e8a35360b17](https://github.com/jaclynbeck-sage/agora-data-tools/commit/5c5dc71e9f5b92ab8b226faaf2877e8a35360b17).